src/API/Globals.php

```diff
@@ -34,6 +35,7 @@ public function __construct(
         private readonly LoggerProviderInterface $loggerProvider,
         private readonly EventLoggerProviderInterface $eventLoggerProvider,
         private readonly TextMapPropagatorInterface $propagator,
+        private readonly HookManager $hookManager,
```

Comment:
hook manager may not need to be globally registered, since all auto-instrumentations are registered at the same time and we probably don't need to access the hook manager again.

Reply:
removed from sdk and globals